### PR TITLE
Use pagy instead of kaminari (in progress) #914

### PR DESCRIPTION
**Pull request by rebecarancan**
Issue:  #914
When I use the Pagy Extra for ElasticSearch, the Active Mode ([https://ddnexus.github.io/pagy/extras/elasticsearch_rails](url)), and try to make a search, I get this error:
Captura de tela de 2019-12-05 18-17-26

I also tried to use the Passive mode, and got the same error.

### CODE REVIEW

**Comment by rebecarancan** [comment]
@sabrina-ulbrich

File: config/initializers/pagy.rb [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+#                  pluralize: lambda{|count| ... } )
+
+
+# I18n extra: uses the standard i18n gem which is ~18x slower using ~10x more memory
+# than the default pagy internal i18n (see above)
+# See https://ddnexus.github.io/pagy/extras/i18n
+# require 'pagy/extras/i18n'
+
+# Default i18n key
+# Pagy::VARS[:i18n_key] = 'pagy.item_name'   # default
```
Comment (wenderjean) [inline config/initializers/pagy.rb]: You have a line ending problem here.

File: app/controllers/profiles_controller.rb [hunk 2]
```diff
@@ -207,8 +203,8 @@ def profiles_for_category
   def profiles_for_search
     Profile.is_published
            .includes(:taggings, :translations)
-           .search(params[:search], params[:filter_countries], params[:filter_cities], params[:filter_lang])
-           .page(params[:page]).per(24)
+           .pagy_search(params[:search])
```
Comment (zaziemo) [inline app/controllers/profiles_controller.rb]: @rebecarancan the problem you are encountering might be based on that line here. The `search` method expects 4 arguments `(params[:search], params[:filter_countries], params[:filter_cities], params[:filter_lang])` but you are only providing the first one. So I think adding them might solve the problem.

**Comment by zaziemo** [comment]
Hey @rebecarancan, I suppose you are not working on this anymore. Therefore I close this PR. If you resume working on this issue, you can reopen it.

**Pull request closed without merging**